tests/integration/conftest.py

```diff
@@ -783,6 +783,9 @@ def prepare_hiveserde_tables(context, random, schema, table_base_dir) -> dict[st
 def prepare_regular_tables(context, external_csv, schema) -> dict[str, TableInfo]:
     tables: dict[str, TableInfo] = {
         "src_managed_table": context.make_table(schema_name=schema.name),
+        "src_managed_non_delta_table": context.make_table(
```

Comment:
do we need to change anything else to verify that integration tests would work correctly?

Reply:
No. I check this additional was covering the scenario for non delta dbfs table. screenshot below
table created
<img width="496" alt="image" src="https://github.com/databrickslabs/ucx/assets/105197202/deda9e9d-1acc-4bcb-8740-5ed10273985b">

screenshot from migrate non delta dbfs workflow task log file
<img width="1466" alt="image" src="https://github.com/databrickslabs/ucx/assets/105197202/91029e22-5e60-460c-9e67-56ad8eff1d1d">

Reply:
Due to the change in https://github.com/databrickslabs/ucx/pull/1623, i had to add the following code in the test_table_migration_job_refreshes_migration_status as it was not updating the migration_status for the last view migrated.
ctx.migration_status_refresher.reset()
    ctx.migration_status_refresher.snapshot()

Reply:
I've fixed this in a separate PR #1630